src/relay/api/resources.py

```diff
@@ -440,12 +440,12 @@ def post(self, args):
                 400,
                 f"Invalid delegation fees: fees too low or not supported currency network of fees",
             )
-        except InvalidMetaTransactionException:
-            abort(400, "The meta-transaction is invalid")
-        except InvalidIdentityContractException:
+        except InvalidMetaTransactionException as exception:
+            abort(400, f"The meta-transaction is invalid: {exception}")
```

Comment:
I think it is not good to print out an exception to a user, it might contain information we dont want to give out.